Directory.Build.targets

```diff
@@ -4,12 +4,21 @@
         <FrameworkVersionRuntime>6.0.0</FrameworkVersionRuntime>
         <FrameworkVersionTesting>6.0.11</FrameworkVersionTesting>
         <YarpVersion>2.0.1</YarpVersion>
```

Comment:
@josephdecock Hi, it seems that net6, net7 targets don't get newer Yarp version. Is it intentional? Wouldn't 2.1.0 work fine here?

Reply:
Yeah, it should work, but not updating the dependency is intentional. Our usual approach to dependencies is to take the earlier version to avoid dependency problems and give users flexibility (your BFF can update to 2.1 explicitly).